src/lib/testrail.ts

```diff
@@ -45,21 +46,32 @@ export class TestRail {
     if (this.options.typeId) {
       url += `&type_id=${this.options.typeId}`
     }
-    return this.makeSync(
-      axios({
-        method:'get',
-        url: url,
-        headers: { 'Content-Type': 'application/json' }, 
-        auth: {
+    let cases = []
+    let previousUrl = null
+    while (url != previousUrl) {
+      previousUrl = url;
+      this.makeSync(
+        axios({
+          method: 'get',
+          url: url,
+          headers: {'Content-Type': 'application/json'},
+          auth: {
             username: this.options.username,
             password: this.options.password
-        } 
-      })
-      .then(response => {
-        return response.data.cases.map(item =>item.id)
-      })
-      .catch(error => console.error(error))
-    )
+          }
+        })
+          .then(response => {
+            cases = cases.concat(response.data.cases.map(item => item.id))
+            if (response.data._links.next) {
+              url = `${this.base}${apiVersionPath}${response.data._links.next}`
```

Comment:
```suggestion
              url = `${this.base}${response.data._links.next}`
```

the "next link" already contains the api version path